docs/source/quick_start.rst

```diff
@@ -0,0 +1,181 @@
+
+
+..
+ Nota bene:  the link for "Latest" should be stable from one release to the next, but check periodically to be sure 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/latest>`
+
+.. code-block:: sh
+  
+  # Uncomment according to the type of file you've downloaded (zip or tar)
+  unzip kokkos-x.y.z.zip 
+  # tar -xzf kokkos-x.y.z.tar.gz
+  cd kokkos-x.y.z
+  mkdir build
+  cd build
+
+
+Basic Configuration
+-------------------
+
+.. note::
+
+  You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make -j`` to build, and ``make install`` to install.
+
+
+
+OpenMP (CPU Parallelism)
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: sh
+
+  cmake \
+   -DKokkos_ENABLE_OPENMP=ON \
+   -DCMAKE_BUILD_TYPE=Release ..
+
+
+.. note::
+
+  Kokkos will attempt to autodetect GPU microarchitecture, but it is also possible to specify the desired `GPU architecture <https://kokkos.org/kokkos-core-wiki/keywords.html#gpu-architectures>`_.  In scenarios where a backend (e.g., CUDA, HIP) is enabled, Kokkos will execute serially on the "host" (CPU).  
+
+CUDA (CPU and GPU)
+~~~~~~~~~~~~~~~~~~
+
+.. code-block:: sh
+
+  cmake \
+    -DKokkos_ENABLE_CUDA=ON \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+
+HIP (CPU and GPU)
+~~~~~~~~~~~~~~~~~
+
+.. code-block:: sh
+
+  cmake \
+    -DKokkos_ENABLE_HIP=ON \
+    -DCMAKE_CXX_COMPILER=hipcc \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+
+Basic Configuration Examples for Spack Builds
+----------------------------------------------
+
+|br|
+Diverse Kokkos variants can be built / installed via Spack.  You will need to select a variant for the desired backend, and appropriate GPU microarchitecture.  To explore the range of variants for a package, ``spack info kokkos``, ``spack info trilinos``, etc.  Please see basic `Spack installation  <https://spack.readthedocs.io/en/latest/getting_started.html>`_ instructions if you're new to the package manager.
+|br|
+|br|
+
+
+.. note::
+
+  Before installing, you can ``spack spec``  variants to verify the build type.
+
+Serial
+~~~~~~~
+
+.. code-block:: sh
+
+  spack spec kokkos@4.2 %gcc@10.3.0 +serial cxxstd=20
+
+OpenMP
+~~~~~~
+
+.. code-block:: sh
+
+  spack spec kokkos@4.2 %gcc@10.3.0 +openmp cxxstd=20
+
+
+CUDA
+~~~~
+
+.. code-block:: sh
+  
+  spack spec / install kokkos@4.2 %gcc@12.2.0 +cuda cuda_arch=70 cxxstd=20 +cuda_relocatable_device_code
+
+
+HIP
+~~~
+
+.. code-block:: sh
+
+  spack spec / install kokkos@4.2 %gcc@12.2.0 +rocm amdgpu_target=gfx90a cxxstd=20
+
+
+Building and Linking a Kokkos "Hello World"
+-------------------------------------------
+
+.. note::
+
+  You will need to set ``Kokkos_ROOT``, and also the root directory for you target backend SDK (i.e., ``CUDA_ROOT``, ``ROCM_PATH``).  Please see `Build, Install and Use <https://kokkos.org/kokkos-core-wiki/building.html>`_ for additional details.
```

Comment:
Not always for backends, they can be in various default paths or loaded via module. We do not talk about that when building Kokkos, why mention it now?
We can also use `CMAKE_PREFIX_PATH` or even `Kokkos_DIR` with CMake.

Reply:
I want to keep the CMake part of this as simple as possible, and dealing with things like `CMAKE_PREFIX_PATH` gets you into more complicated CMake territory (requiring and understanding of CMake modules).  I can fix the language to say these variables need to be set, no matter who sets them (i.e., system modules, or the user).